zkSNACKs/WalletWasabi
# Start Wasabi on Linux distributions boot up

Part of #5956

On Ubuntu 20.04 we need to create a `Wasabi.desktop` file under `home/your_username/.config/autostart/` to start WW with the OS.

**How to test this PR**

1. Get this PR.
2. In `General Settings` fiddle with the `Start Wasabi with the system` option.
3. See that the `.desktop` file gets created/deleted in `home/your_username/.config/autostart/` folder.
4. Restart your PC and see if WW indeed launches after logging in.

**ToDo**
- [x] Test this on older Ubuntus.
- [x] Test this on Debian 10.
- [x] Test this on Fedora.
